packages/react-renderer/package.json

```diff
@@ -28,24 +33,28 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/honojs/middleware.git"
+    "url": "git+https://github.com/honojs/middleware.git",
+    "directory": "packages/react-renderer"
   },
   "homepage": "https://github.com/honojs/middleware",
   "peerDependencies": {
-    "hono": "*"
+    "hono": "*",
+    "react": "*",
+    "react-dom": "*"
```

Comment:
`tsup` will automatically exclude any `peerDependencies` from the bundle

It may be worth updating these to something more useful? eg;

```suggestion
    "hono": "*",
    "react": "^18 || ^19",
    "react-dom": "^18 || ^19"
```

Reply:
It might be better to specify it, but there haven't been any problems so far, so let's leave it as it is.

Reply:
Should I remove `react` and `react-dom` from peer dependencies, or leave them as `*`?

Reply:
Can you leave them as `*` and don’t remove them?

Reply:
Sure thing 👍🏻